### ISSUE TITLE
how to use qnn.BatchNorm2dToQuantScaleBias?

### ISSUE THREAD

**Issue by ardeal** [Xilinx/brevitas#711]
Hi,

if I use `qnn.BatchNorm2dToQuantScaleBias` in my network, the training loss remains around 5.15 and will not get smaller. The traid model performs badly.

If I use `nn.BatchNorm2d` instead, the training loss gets smaller repidly. The trained model performs very good.

is there any doc aboud how to use `qnn.BatchNorm2dToQuantScaleBias`? or is there any example code about how to use `qnn.BatchNorm2dToQuantScaleBias`?

```python

self.conv1 = qnn.QuantConv2d(in_channels, cfg[0], kernel_size=(3,3), padding=1, input_quant=qfloat8, weight_bit_width=8)
# self.bn1   = qnn.BatchNorm2dToQuantScaleBias(cfg[0], input_quant=Int8BiasPerTensorFloatInternalScaling, weight_bit_width=8)
self.bn1 = nn.BatchNorm2d(cfg[0])
self.relu1 = qnn.QuantReLU(bit_width=8)
self.pool1 = qnn.QuantMaxPool2d(kernel_size=2, stride=2)

self.conv2 = qnn.QuantConv2d(cfg[0], cfg[2], kernel_size=3, padding=1, input_quant=qfloat8, weight_bit_width=8)

```
